testdata/p4_16_samples/issue4901_new_start_with_start_state_loopback.p4

```diff
@@ -0,0 +1,23 @@
+parser Parser1() {
+  bit<8> l = 0;
+  state start {
+    transition next;
+  }
+  state next {
+    transition select(l) {
+        2 : start;
+        default: accept;
+    }
+  }
+}
+
+parser Parser2() {
+  state start {
+    transition  accept;
+  }
+}
```

Comment:
I'd add some initializer here just to see that there is no confusion between the two parsers.

Reply:
I'd prefer to not change this test (it is the case taken from https://github.com/p4lang/p4c/issues/4901) but I added a new one.